automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -132,6 +130,31 @@ class RawlsApiSpec extends TestKit(ActorSystem("MySpec")) with FreeSpecLike with
       }
     }
 
+    "should create and destroy a workflow collection resource in Sam for a workspace" in {
```

Comment:
wouldn't that only test that i'd called the functions i just typed? imo this way tests the behaviour, which is better... plus it's resilient to implementation changes, like moving to `createResourceFull`

Reply:
this certainly tests more, namely that the sam dao is right, but integration tests are less accessible than unit tests. Where I think we should get is to use swagger to generate a sam client lib. Use that client lib in the sam integration level tests. Use the client libs in rawls knowing they are well tested. Then this should be a unit test and ease of use trumps behavior testing since we don't actually want to test sam's behavior.

but we are not there yet so I will let it go.